distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -975,13 +975,15 @@ type Integer
 
             Integer.parse "20220216"
     parse : Text -> Text -> Integer ! Parse_Error
-    parse text (radix=10) =
-        Panic.catch NumberFormatException (Long.parseLong text radix) _->
-             Error.throw (Number_Parse_Error.Error text)
+    parse text (radix=10) = Integer.parse_internal text radix
+
+    parse_internal text radix = @Builtin_Method "Integer.parse"
 
 ## UNSTABLE
 
+
    A syntax error when parsing a double.
```

Comment:
Why another newline here?

I guess we are usually doing
```suggestion
## UNSTABLE

   A syntax error when parsing a double.
```
or
```suggestion
## UNSTABLE
   A syntax error when parsing a double.
```
 but I don't recall two empty lines there.

(We should probably standarize if there's 1 or 0 too...)